RecoParticleFlow/PFClusterProducer/plugins/SimMappers/GenericSimClusterMapper.cc

```diff
@@ -43,7 +43,7 @@ buildClusters(const edm::Handle<reco::PFRecHitCollection>& input,
     reco::PFCluster& back = output.back();
     edm::Ref<std::vector<reco::PFRecHit> > seed;    
     double energy = 0.0, highest_energy = 0.0;
-    auto hitsAndFractions = std::move( sc.hits_and_fractions() );
+    auto hitsAndFractions = sc.hits_and_fractions() ;
```

Comment:
In both cases where I removed `std::move` the clang compiler warning was:
```
warning: moving a temporary object prevents copy elision [-Wpessimizing-move]
     auto hitsAndFractions = std::move( sc.hits_and_fractions() );
```

i.e. in both cases it was a temporary anyway so `std::move` was not useful even if copy elision is not available.